app/views/posts/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @posts.each do |post| %>
```

Comment:
It would be nice to have login (if not logged in) and logout (if logged in) links at the top of this page.